docs/configuration.md

```diff
@@ -47,3 +47,5 @@ SENTRY_DSN='https://user:pw@sentry.io/1234' #OPTIONAL
 LOG_FORMAT=json # OPTIONAL
 # By default, logs are formatted for readability in development. If you intend to drain logs to a logging service, use this option.
 ```
+
+For more information on the formatting conventions and rules of `.env` files, check out [the npm documentation](https://www.npmjs.com/package/dotenv#rules).
```

Comment:
"the npm documention" makes me think it's a link to the docs for `npm` itself. Perhaps something like "the `dotenv` module's docs" would be more clear.